styles.css

```diff
@@ -0,0 +1,501 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@300&family=Poppins:wght@100&family=Roboto&display=swap");
+
+body {
+  background-color: #e5e5e5;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  min-width: 327px;
+}
+
+a {
+  text-decoration: none;
+}
+
+.mobile {
+  background-color: #fff;
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.image,
+.banner,
+.works {
+  align-self: center;
+  width: 327px;
+}
+
+.head {
+  height: auto;
+  align-self: center;
+  padding-top: 40px;
+}
+
+h1 {
+  text-align: center;
+}
+
+.title {
+  height: 50px;
+  width: 327px;
+  color: #1c1a19;
+  font-family: Crete Round, serif;
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 52px;
+  letter-spacing: 0;
+  text-align: left;
+  border-radius: 0;
+  align-self: center;
+  margin-left: 15px;
+}
+
+.head hr {
+  border-bottom: 2px solid slateblue;
+  width: 90%;
+  height: 0;
+  margin-left: 5%;
+  margin-right: 5%;
+}
+
+.banner {
+  color: #091e42;
+  height: 360px;
+  display: flex;
+  flex-direction: column;
+}
+
+.image {
+  height: 237px;
+  border-radius: 0;
+  background-image: url("assets/images/ImgPlaceholder.png");
+  background-repeat: no-repeat;
+  background-size: 98% 93%;
+  padding-top: 20px;
+  margin-left: 1%;
+}
+
+.banner h2 {
+  height: 36px;
+  font-family: Crete Round, serif;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 44px;
+  letter-spacing: 0;
+  text-align: left;
+  margin-left: 20px;
+}
+
+.banner p {
+  height: 147px;
+  font-family: Inter, serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+  margin-left: 20px;
+}
+
+.pr-button {
+  border: none;
+}
+
+.project {
+  background-color: #ff6b00;
+  height: 34px;
+  width: 100px;
+  border-left: 20px;
+  border-bottom: 10px;
+  border-radius: 0;
+  color: #fff;
+  text-decoration: none;
+  font-family: Inter, serif;
+  font-size: 17px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: center;
+  border: none;
+}
+
+.see-pr {
+  margin-left: 25px;
+}
+
+.bn-buttons {
+  width: auto;
+  height: 20px;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  padding-left: 0;
+  font-family: Inter, serif;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 18px;
+  letter-spacing: 0;
+  text-align: left;
+}
```

Comment:
- [ ] Did you know that the Figma design provides you exactly how buttons should look like? You should consider changing them a little to match the original design better, you can check it here:

Figma design:
![image](https://user-images.githubusercontent.com/66967274/144603342-69d7a76f-999b-4071-be6e-c8d46923c9ca.png)

Your design:

![image](https://user-images.githubusercontent.com/66967274/144603408-08710212-7e12-4798-844f-dfb8c3cc5e2f.png)

Reply:
Thank you, Vanessa. 

I tried to make it as close to the original design as possible, taking into account the peculiarities of the text length. 

I know that there is no limit to perfection, but I tried my best.